### PR TITLE
Noviny/add prettier on changelog write

### PR DESCRIPTION
**Pull request by Noviny**

### CODE REVIEW

File: packages/cli/src/utils/updateChangelog/index.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-
 import path from "path";
 import util from "util";
+import prettier from "prettier";
+
 import generateMarkdownTemplate from "./template";
 import logger from "../logger";
 import * as bolt from "../bolt-replacements";
 
+prettier.resolveConfig(process.cwd());
```
Comment (mitchellhamilton) [inline packages/cli/src/utils/updateChangelog/index.js]: What's this for?

Comment (Noviny) [reply packages/cli/src/utils/updateChangelog/index.js]: This gets the local prettier config, so we don't accidentally fight whatever their prettier settings are.

Comment (mitchellhamilton) [reply packages/cli/src/utils/updateChangelog/index.js]: Does it magically set the options or are you meant to use the return value?

Comment (Noviny) [reply packages/cli/src/utils/updateChangelog/index.js]: No, I just misread the API docs.

**Pull request merged**